Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Use this with
+#
+#  docker build -t evm_rpc .
+#  or use ./scripts/docker-build
+#
+# The docker image. To update, run `docker pull ubuntu` locally, and update the
+# sha256:... accordingly.
+FROM ubuntu@sha256:626ffe58f6e7566e00254b638eb7e0f3b11d4da9675088f4781a50ae288f3322 as deps
+
+ENV TZ=UTC
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
+    apt -yq update && \
+    apt -yqq install --no-install-recommends curl ca-certificates \
+        build-essential pkg-config libssl-dev llvm-dev liblmdb-dev clang cmake
+
+# Install Rust and Cargo in /opt
+ENV RUSTUP_HOME=/opt/rustup \
+    CARGO_HOME=/cargo \
+    PATH=/cargo/bin:$PATH
+
+WORKDIR /evm_rpc
```

Comment:
unless I'm missing something, I don't think you need an explicit working directory, the default (I think `/`) should be fine

Reply:
This is a workaround for a nebulous error message (along the lines of `ERROR: cannot copy to non-directory: /var/lib/docker/overlay2/fso8ezwpr886l9bjnx3jni37e/merged/lib`). I'll address this in a follow-up PR (trying to get this merged by the end of the day if possible).